### ISSUE TITLE
Printing long tables causes repeated header to overlap with content

### ISSUE THREAD

**Issue by timvink** [squidfunk/mkdocs-material#1939]
__I checked that...__

- [x] ... the documentation does not mention anything about my problem
- [x] ... the problem doesn't occur with the default MkDocs template
- [x] ... the problem is not in any of my customizations (CSS, JS, template)
- [x] ... there are no open or closed issues that are related to my problem

## Description

When printing tables that are longer than 1 page, the header is repeated but overlaps with the content.

### Expected behavior

Repeating the header is fine, but it should not overlap with content.

### Actual behavior

It's overlapping:

![image](https://user-images.githubusercontent.com/5570380/93879095-6c1b7e80-fcdb-11ea-81c3-855a51ac20ce.png)

### Steps to reproduce the bug

Insert a long markdown table into a page and go to File > Print and look at the print preview.

I made a reproducible, minimal example. View it with:

```bash
git clone git@github.com:timvink/mkdocs-print-site-plugin.git
cd mkdocs-print-site-plugin
git checkout 7d787b5902cabbcf3c5cf9ffce85e64eb150b2fd
mkdocs serve -f tests/fixtures/projects/tables/mkdocs.yml
```

### Package versions

Python 3.7.4
mkdocs, version 1.1.2
Material Version: 5.5.12

### System information

* OS: MacOS
* Browser: Chrome

**Comment by squidfunk** [comment]
Does this happen for other browsers or only Chrome?

**Comment by timvink** [comment]
Works in MacOS / Safari

![image](https://user-images.githubusercontent.com/5570380/93890252-71cc9080-fcea-11ea-9f57-d5d4cf76259e.png)

And in MacOS / Firefox there is actually a different issue, where the table is cut-off and limited to only 1 page:

![image](https://user-images.githubusercontent.com/5570380/93890844-0e8f2e00-fceb-11ea-811d-150927a0164d.png)

**Comment by timvink** [comment]
Yeah, this seems a very tricky issue. But it might be possible to fix, as `mkdocs` base theme seems to have got it right (they just forgot to remove the sidebar is print mode):

macOS / chrome

![image](https://user-images.githubusercontent.com/5570380/93919302-c5040a80-fd0d-11ea-8015-cb90def724f7.png)

macOS / safari

![image](https://user-images.githubusercontent.com/5570380/93919392-e533c980-fd0d-11ea-92e9-ab04502742c1.png)

macOS / firefox

![image](https://user-images.githubusercontent.com/5570380/93919461-04325b80-fd0e-11ea-88c4-75579d50529d.png)

**Comment by squidfunk** [comment]
The problem was that the table was wrapped in an `inline-block` element, which was necessary to make them horizontally scrollable. b9ce0862 now resets this on `print`. Can you check the latest `master` and confirm that the issue is gone?

**Comment by timvink** [comment]
Solved, nice 🎉 ! Thanks for all your support & hard work!

![image](https://user-images.githubusercontent.com/5570380/94007657-14941600-fda2-11ea-9a3f-90938098abf1.png)

**Comment by squidfunk** [comment]
Released as part of 5.5.14.
